### ISSUE TITLE
SHIFT + Click didn't work

### ISSUE THREAD

**Issue by Quorafind** [zsviczian/obsidian-excalidraw-plugin#80]
**Describe the bug**
When I pressed SHIFT and click on a text, nothing happened.

**To Reproduce**
1. Type the filename of file in the excalidraw page
2. Shift + Click on it
3. Nothing happened(but the ctrl+click worked

**Expected behavior**
Shift + Click works, and the note would open in a new tab.

**Screenshots**
![image](https://user-images.githubusercontent.com/13215013/122702099-4549a300-d281-11eb-8884-718d88a4683a.png)

https://user-images.githubusercontent.com/13215013/122702185-6d390680-d281-11eb-9339-7c69fb98d3a8.mp4

**Desktop (please complete the following information):**
 - OS: Windows
 - Version: 0.1.16

**Comment by zsviczian** [comment]
I am sorry to hear that. 
I can't reproduce the error. I see you seem to be using sliding panes. I tried it with sliding panes as well, and it worked fine.

CTRL+click to open the file
CTRL+ALT+click to create the file (if it does not yet exist) and open it
CTRL+SHIFT+click to open the file in a new pane
CTRL+ALT+SHIFT+click to create the file (if it does not yet exist) and open it in a new pane

I tried them all, it seem to work for me well.

What happens when you use the link button?
![image](https://user-images.githubusercontent.com/14358394/122713179-9186ea00-d265-11eb-919b-9b74daf1f6a6.png)
